configure_check_conditions.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>Now we will use a Suite to</p><p> </p><p>automatically</p><p> </p><p>**and general fix - elaborate shortly about the fact that the conditions results can be run/displayed only within a suite</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>We are running them in suite and displaying them automatically but it's possible to run on a single check <code>check.conditions_decision(check.run(dataset))</code></p>